pkg/utils/docker_error.go

```diff
@@ -21,9 +21,24 @@ type DockerError struct {
 }
 
 const (
-	errOpValidate         = "docker_validate" // validate docker images
-	errOpClientCreate     = "CLIENT_CREATE_ERROR"
-	errOpContainerInspect = "CONTAINER_INSPECT_ERROR"
+	errOpValidate            = "DOCKER_VALIDATE"
+	errOpClientCreate        = "CLIENT_CREATE_ERROR"
+	errOpContainerInspect    = "CONTAINER_INSPECT_ERROR"
+	errOpContainersStopped   = "CONTAINERS_STOPPED"
```

Comment:
is there a reason this is plural (the rest are singular) and doesn't include the suffix `ERROR`? This and `errOpStopContainer       = "CONTAINER_STOP_ERROR"` are very similar, could these be put into one?

Reply:
No reason, good spot